.qt_for_python/uic/logger_remote.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'c:\Users\b16lu\pylabnet\pylabnet\gui\pyqt\gui_templates\logger_remote.ui'
+#
```

Comment:
@pieterjanstas , what exactly is this file? Seem to be automatically generated.

Reply:
Yeah I think it was was created when I changed the .ui file in Qt. I think I can delete it.